pkg/environment/api/project.go

```diff
@@ -120,8 +120,12 @@ func (s *EnvironmentService) ListProjects(
 		return nil, err
 	}
 	whereParts := []mysql.WherePart{}
-	if req.OrganizationId != "" {
-		whereParts = append(whereParts, mysql.NewFilter("organization_id", "=", req.OrganizationId))
+	if req.OrganizationIds != nil && len(req.OrganizationIds) > 0 {
+		oIDs := make([]interface{}, 0, len(req.OrganizationIds))
```

Comment:
Please refactor this by creating a new function `convToInterfaceSlice`.


https://github.com/bucketeer-io/bucketeer/blob/0685cc546e86fb587be05f83b71303b066fc4a34/pkg/feature/api/feature.go#L1770-L1778

Reply:
Added `convToInterfaceSlice()`and modified it to convert Organization IDs using that function.
https://github.com/bucketeer-io/bucketeer/pull/810/commits/b462fb992c254e49d5c302a460712469e9ffc006